pkg/render/istio/istio.go

```diff
@@ -109,6 +110,7 @@ func Istio(cfg *Configuration) (*IstioComponentCRDs, *IstioComponent, error) {
 		IstioCNIOpts: IstioCNIOpts{
 			Global: &GlobalConfig{
 				IstioNamespace: IstioNamespace,
+				NativeNftables: ptr.BoolToPtr(true),
```

Comment:
Just to I understand, this configuration only applies to the rules programmed within the application pod (to send traffic to ztunnel) and not host rules, right? 

If so I think it's OK to set this to true unconditionally, otherwise we'll at least need to verify compatibility with the mode Felix is using.

Reply:
sorry i should've opened this as Draft, used as a discussion item for something @hjiawei and I are looking at right now


but yes this is the rules in the application pod only so it's quite safe